server/router.go

```diff
@@ -26,6 +26,10 @@ type Router struct {
 	srv  *grpc.Server
 }
 
+func (r *Router) EndSession(ctx context.Context, request *net.EndSessionRequest) (*net.EndSessionResponse, error) {
+	panic("implement me")
```

Comment:
I think it could fine for this to be a no-op i.e. just return `&net.EndSessionRequest, nil` because the way the router is supposed to work is:

- B sends GetOrchestrator to router
- Router forwards GetOrchestrator to all available Os and returns the first OrchestratorInfo response to B
- The OrchestratorInfo.Transcoder will be the URL used by the B for subsequent segment requests
- So, a EndSessionRequest from B will go to OrchestratorInfo.Transcoder and not the router

If this is the case, then the router wouldn't need to handle the EndSessionRequest and could likely just no-op.

TBH I don't know if anyone actually uses this as it was experimental, but if the above addresses how to handle this then can punt what we want to do with this router concept...